client/tailscale/tailscale.go

```diff
@@ -108,6 +108,31 @@ func Goroutines(ctx context.Context) ([]byte, error) {
 	return body, nil
 }
 
+// BugReport logs and returns a log marker that can be shared by the customer with support.
+func BugReport(ctx context.Context, note string) ([]byte, error) {
+	var queryString string
+	if note != "" {
+		queryString = "?note=" + note
+	}
+	req, err := http.NewRequestWithContext(ctx, "GET", "http://local-tailscaled.sock/localapi/v0/bugreport"+queryString, nil)
```

Comment:
"GET" isn't quite right for a mutating action. POST?

Reply:
bad copy paste. done